web/src/app/validation/validation-util.ts

```diff
@@ -19,4 +19,14 @@ export class ValidationUtil {
         }
         return true;
     }
+
+    public static compareStrTrimed(s1: string, s2: string): boolean {
+        if (s1 === undefined && s2 === undefined || s1 === null && s2 === null) {

```

Comment:
Please remove the first check and replace the ```return false``` in the end by ```return s1 === undefined && s2 === undefined || s1 === null && s2 === null```. This way, we only need one check in case all strings are set.